src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/PermalinkController.cs

```diff
@@ -7,7 +8,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Api.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/permalink")]
```

Comment:
Why this change out of interest?  Are we generally switching to this convention now?

Reply:
I felt like it's safer to define the route string explicitly as the path mustn't change unless there's corresponding changes to the FE.

I think that the replacement of the `[controller]` token for the class name into the route is nice but is vulnerable to the class names changing during a refactor.

After this change we are down to 5 controllers  still using the token.

Reply:
Yeah I much prefer this way tbh.  If for no other reason, it's quick at a glance to see all code that refers to that route, both in the front end and back.